modules/tournament/src/main/PlayerRepo.scala

```diff
@@ -25,6 +24,14 @@ final class PlayerRepo(coll: Coll)(implicit ec: scala.concurrent.ExecutionContex
 
   def byId(id: Tournament.ID): Fu[Option[Player]] = coll.one[Player](selectId(id))
 
+  def tournamentsByUserId(userId: User.ID): Fu[List[Tournament.ID]] =
+    coll.primitive[Tournament.ID](
+      $doc(
+        "uid" -> userId
+      ),
+      "tid"
+    )
```

Comment:
two problems here
- this fetches all tournament IDs the user has ever played, it will be thousands for many players
- there's no DB index on the `uid` field and adding it would cost a few RAM gigabytes